src/applications/disability-benefits/2346/sass/form-2346.scss

```diff
@@ -53,12 +53,12 @@ label {
 }
 
 .required {
-  color: $color-secondary-dark;
+  color: red;
   font-weight: normal;
 }
 
 [type=checkbox]:checked + label::before, [type=radio]:checked + label::before {
-   background-color: $color-white;
+   background-color: #fff;
```

Comment:
https://github.com/department-of-veterans-affairs/vets-website/pull/11999/files#r406826151